docs/MO_DG/Deep_Learning_Model_Optimizer_DevGuide.md

```diff
@@ -81,54 +77,50 @@ mo --help
 
 ## Examples of CLI Commands
```

Comment:
```suggestion
## Example Commands of Using Model Optimizer
```

Reply:
I think the "Examples of CLI Commands" is better option. If we really want to change it, then how about this:

```suggestion
## Example Commands of Model Optimizer to Use
```

Reply:
I am also an advocate of the phrase "Examples of CLI Commands". IMHO it is self-explanatory and does not need  elaboration.

Reply:
I would agree the article itself indicates that CLI commands refer to MO. In this context it should be fine to leave the original version.

Reply:
```suggestion
## Examples of CLI Commands
```